docs/root/operations/admin.rst

```diff
@@ -174,3 +174,21 @@ The fields are:
   .. http:get:: /runtime?format=json
 
   Outputs /runtime in JSON format. This can be used for programmatic access of runtime values.
+
+.. http:get:: /hystrix_event_stream
+
+  Start streaming :ref:`event stream<https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events>` to hystrix dashboard. 
```

Comment:
Nit: s/hystrix/Hystrix/

Reply:
Thanks for all your comments. Will update everything with new commit, also DCO was missing.. In any case, this depends on https://github.com/envoyproxy/envoy/pull/2736 which currently is blocked by failing checks (taking care of this as well)